export class ErrorShield {
  catch(error: any, context: { req: any, res: any }) {
    
  }
}
